apps/api-extractor/src/generators/ApiModelGenerator.ts

```diff
@@ -653,14 +653,16 @@ export class ApiModelGenerator {
       const apiItemMetadata: ApiItemMetadata = this._collector.fetchApiItemMetadata(astDeclaration);
       const docComment: tsdoc.DocComment | undefined = apiItemMetadata.tsdocComment;
       const releaseTag: ReleaseTag = apiItemMetadata.effectiveReleaseTag;
+      const isReadonly: boolean = this._determineReadonly(astDeclaration);
```

Comment:
My understanding is that only the `processApiVariable`, `processApiProperty`, `processApiMethod`, `processApiPropertySignature`, and `processApiMethodSignature` methods should need to call `_determineReadonly`, because those are the only entities that can possibly `isReadonly` set to true. What am I missing?

Reply:
Reference below on the mixup